test/unit/JestConfigEditorSpec.ts

```diff
@@ -40,12 +45,20 @@ describe('JestConfigEditor', () => {
     assert(defaultConfigLoaderStub.loadConfig.calledOnce, 'DefaultConfigLoader loadConfig not called');
   });
 
-  it('should call the reactConfigLoader loadConfig method when no project is defined', () => {
+  it('should call the ReactScriptsJestConfigLoader loadConfig method when no project is defined', () => {
     config.set({ jest: { project: 'react' } });
 
     jestConfigEditor.edit(config);
 
-    assert(reactConfigLoaderStub.loadConfig.calledOnce, 'ReactConfigLoader loadConfig not called');
+    assert(reactScriptsJestConfigLoaderStub.loadConfig.calledOnce, 'ReactScriptsJestConfigLoader loadConfig not called');
+  });
+
+  it('should call the ReactScriptsTSJestConfigLoader loadConfig method when no project is defined', () => {
```

Comment:
But a project is defined in the test

Reply:
Will update the tests, it seems their description is not correct atm.